lib/v2/dlnews/category.js

```diff
@@ -0,0 +1,69 @@
+const cheerio = require('cheerio');
+const got = require('@/utils/got');
+const { getData, getList } = require('./utils');
+
+const _website = 'dlnews';
+const topics = {
+    defi: 'DeFi',
+    fintech: 'Fintech/VC/Deals',
+    'llama-u': 'Llama U',
+    markets: 'Markets',
+    'people-culture': 'People & Culture',
+    regulation: 'Regulation',
+    snapshot: 'Snapshot',
+    web3: 'Web3',
+};
+
+module.exports = async (ctx) => {
+    const { category } = ctx.params;
+    const baseUrl = 'https://www.dlnews.com';
+    const apiPath = '/pf/api/v3/content/fetch/articles-api';
+    let vertical;
+    if (category) {
+        vertical = String(category);
+    } else {
+        vertical = '';
+    }
+
+    const query = {
+        author: '',
+        date: 'now-1y/d',
+        offset: 0,
+        query: '',
+        size: 15,
+        sort: 'display_date:desc',
+        vertical,
+    };
+    const data = await getData(`${baseUrl}${apiPath}?query=${encodeURIComponent(JSON.stringify(query))}&_website=${_website}`);
+    const list = getList(data);
+    const items = await Promise.all(
+        list.map((item) =>
+            ctx.cache.tryGet(item.link, async () => {
+                const { data: response } = await got(item.link);
+                const $ = cheerio.load(response);
+                item.description = $('.article-body-wrapper')
+                    .clone()
+                    .children('div.move-to-right')
+                    .remove()
+                    .end()
+                    .children('div#comments-form-wrapper')
+                    .remove()
+                    .end()
+                    .map((index, element) => $(element).html())
+                    .get()
+                    .join('');
```

Comment:
Please use the embedded `Fusion.globalContent` object instead.
docs: https://docs.rsshub.app/en/joinus/new-rss/before-start.html#acquire-data